GPU/TPCSpaceChargeBase/AliTPCSpaceCharge3DCalc.cxx

```diff
@@ -2242,8 +2356,9 @@ void AliTPCSpaceCharge3DCalc::IntegrateDistCorrDriftLineDz(
         /////////////// use irregular grid look up table for correction
         // set
         if (fCorrectionType == kIrregularInterpolator) {
+          // values should be negative of distortions
           (*mCorrIrregularDrDz)(i, j) = -drDist;
-          (*mCorrIrregularDPhiRDz)(i, j) = -1 * dPhi * (radius0 + drDist);
+          (*mCorrIrregularDPhiRDz)(i, j) = -1 * dPhi * radius0;
```

Comment:
The reference points for the irregular correction lookup tables are the distorted points, right? So for the drphi correction, it should be dPhi * the distorted radius, which is radius0 + drDist.
(*mCorrIrregularDPhiRDz)(i, j) = -1 * dPhi * (radius0 + drDist);